coursemapper-kg/recommendation/app/services/course_materials/db/neo4_db.py

```
@@ -1704,6 +1784,948 @@
+        def get_resource_primary_key(resource: dict):
+            return resource["rid"] if "rid" in resource else resource["id"]
+            
+        logger.info("Store Resources: Videos | Articles")
+        # result = []
+
+        tx = self.driver.session()
+        if resources_form == "dict":
+            for key, resources in resources_dict.items():
+                if key == "videos":
+                    if len(resources) > 0:
+                        logger.info(f"Creating Resources YouTube AND Updating Relationship between Resource and Concept_modified: {len(resources)} Resources")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)